### PR TITLE
Added load objects.

### LINKED ISSUE #79: Load objects on level and set the texture
Because objects of abstract level dont need jpg textures, you must set them emission in code. They must looks like on attachment screen. That means they must glows because we not use hdri map on this lvl. For create snake you need use 2 elements 
* SnakePattern_head.obj - this is model was a head of snake that should be in front of the snake as in the screenshot
* SnakePattern_Body.obj - this is snake body, when you load this element in lvl you see this is one square pattern, you need to build a snake body with this pattern, use array of this elements for creating the snake body as the attached screenshot.
image

### LINKED ISSUE #84: Fix buttons in AbstractLvl
Need to fix "Back to menu" button, this must be in left up corner on screen like on the test level.
Fix "tap" button, there is no this button on this lvl now, need to add they on screen.

### LINKED ISSUE #85: Correct camera position in AbstractLvl
Now camera position is incorrect because snake move from left to right. Snake must move from bottom to top. 
And you can see the objects of the snake's body from the side, rotate the camera so that it can be seen from top like on screen in concept.

### PR DESCRIPTION
**Pull request by IgorekLoschinin**
fix #79 Added upload objects.
fix #84 Added change position btn 'baskTomenu'.
fix #85 Added change camera position.

### CODE REVIEW

**Comment by EndrII** [comment]
please update QuasarAppLib to last main commit for fix build

File: src/CrawlAbstractLvl/private/abslvlworld.cpp [hunk 1]
```diff
@@ -25,14 +26,30 @@ CRAWL::IPlayer *AbsLvlWorld::initPlayer() const {
...
+                {registerObject<ObstacleBlue>(), 1}
+            }
+        },
+
+        {300,
+            {
+                {registerObject<ObstacleRed>(), 4}
+            }
+        }
+
```
Comment (EndrII) [inline src/CrawlAbstractLvl/private/abslvlworld.cpp:47]: i think you forget about light object

File: src/CrawlAbstractLvl/private/abslvlworld.cpp [hunk 2]
```diff
@@ -25,14 +26,30 @@ CRAWL::IPlayer *AbsLvlWorld::initPlayer() const {
...
     };
 }
 
 QString AbsLvlWorld::initHdrBackGround() const {
-    return "qrc:/hdr/hdr/plate_absLvl.hdr";
+    return "qrc:/hdr/hdr/testHDR.hdr";
```
Comment (EndrII) [inline src/CrawlAbstractLvl/private/abslvlworld.cpp:52]: please restore old hdr map

**Review by EndrII — CHANGES_REQUESTED** [review]

File: src/CrawlAbstractLvl/private/obstaclered.cpp [hunk 3]
```diff
@@ -7,12 +7,18 @@
 
 #include "obstaclered.h"
 
-ObstacleRed::ObstacleRed() : IWorldItem("ObstacleRed") {
+namespace AbstractLvl {
+
+ObstacleRed::ObstacleRed() : IWorldItem(AUTO_CLASS_NAME) {
     setMash("qrc:/mesh/meshes/ObstacleRed.mesh");
-    setSize({1,1,1});
+    setSize({2,2,2});
```
Comment (EndrII) [inline src/CrawlAbstractLvl/private/obstaclered.cpp:14]:
```suggestion
    setSize({1,1,1});
```
please use the the 1 as a scale factor. if you think that object is small then please create new task for the @Oleg-designer

File: src/CrawlAbstractLvl/private/obstaclered.cpp [hunk 4]
```diff
@@ -7,12 +7,18 @@
...
+    setColor("#ff1927");
+    setposition({300, 0, 200});
```
Comment (EndrII) [inline src/CrawlAbstractLvl/private/obstaclered.cpp:16]:
```suggestion

```
The position of object automatically recalculate in world initialization

**Review by EndrII — CHANGES_REQUESTED** [review]

File: src/Client/languages/tr.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE TS>
 <TS version="2.1">
 <context>
-    <name>AbsLvlWorld</name>
+    <name>AbstractLvl::AbsLvlWorld</name>
```
Comment (EndrII) [inline src/Client/languages/tr.ts:5]: maybe you added wrong name for level

Comment (IgorekLoschinin) [reply src/Client/languages/tr.ts:5]: I changed the name of the class. I FIX.

**Review by EndrII — APPROVED** [review]

**Pull request merged**